### PR TITLE
🌐 Add Chinese translation for `docs/zh/docs/tutorial/background-tasks.md`

### PR DESCRIPTION
**Pull request by maoyibo**
🌐 Add Chinese translation for Tutorial - Background Tasks

### CODE REVIEW

File: docs/zh/docs/tutorial/background-tasks.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+
+然后，在 *路径操作函数* 生成的另一个后台任务将使用 `email` 路径参数写一条消息。
+
+## 技术细节
+
+`BackgroundTasks` 类直接来自于 <a href="https://www.starlette.io/background/" class="external-link" target="_blank">`starlette.background`</a>.
+
+它被直接导入/包含到 FastAPI 中，这样你就可以从 FastAPI 中导入它，避免意外地从 `starlette.background` 中导入替代的 `BackgroundTask` (末尾没有 `s` )。
+
+通过只使用 `BackgroundTasks` (而不是 `BackgroundTask` )，然后就可以使用它作为 *路径操作函数* 参数，并让 **FastAPI** 为您处理其余的，就像直接使用 `Request` 对象一样。
```
Comment (blt232018) [inline docs/zh/docs/tutorial/background-tasks.md:76]:
```suggestion
仅通过使用 `BackgroundTasks`（并不是 `BackgroundTask`），然后就可以把它作为*路径操作函数*参数使用，并让 **FastAPI** 为您处理其余操作，就像直接使用 `Request` 对象时一样。
```

File: docs/zh/docs/tutorial/background-tasks.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+在 FastAPI 中仍然可以单独使用 `BackgroundTask` ，但你必须在代码中创建 `BackgroundTask` 对象并返回一个包含它的Starlette `Response` 。
```
Comment (blt232018) [inline docs/zh/docs/tutorial/background-tasks.md:78]:
```suggestion
在 FastAPI 中仍然可以单独使用 `BackgroundTask`，但你必须在代码中创建 `BackgroundTask` 对象并返回一个包含它的 Starlette `Response`。
```

File: docs/zh/docs/tutorial/background-tasks.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+
+你可以在<a href="https://www.starlette.io/background/" class="external-link" target="_blank">Starlette后台任务的官方文档</a>中看到更多细节。
```
Comment (blt232018) [inline docs/zh/docs/tutorial/background-tasks.md:80]:
```suggestion
你可以在 <a href="https://www.starlette.io/background/" class="external-link" target="_blank">Starlette 后台任务的官方文档</a>中看到更多细节。
```

File: docs/zh/docs/tutorial/background-tasks.md [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+
+## 警告
+
+如果你需要执行较重的后台计算，且不一定需要由相同的进程执行(例如,你不需要共享内存变量,等等)，你可能会受益于使用其他更大的工具,像<a href="https://docs.celeryproject.org" class="external-link" target="_blank">Celery</a>。
```
Comment (blt232018) [inline docs/zh/docs/tutorial/background-tasks.md:84]:
```suggestion
如果你需要执行繁重的后台计算，且不一定需要由同一进程执行（例如，你不需要共享内存、变量等），你可能会受益于使用其他更大型的工具，例如 <a href="https://docs.celeryproject.org" class="external-link" target="_blank">Celery</a>。
```

File: docs/zh/docs/tutorial/background-tasks.md [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+
+它们往往需要更复杂的配置，一个消息/任务队列管理器，如 RabbitMQ 或 Redis ，但它们允许你在多个进程中运行后台任务，特别是在多个服务器中。
```
Comment (blt232018) [inline docs/zh/docs/tutorial/background-tasks.md:86]:
```suggestion
它们往往需要更复杂的配置，一个消息/任务队列管理器，如 RabbitMQ 或 Redis，但它们允许你在多个进程中运行后台任务，特别是在多个服务器中。
```

File: docs/zh/docs/tutorial/background-tasks.md [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+要查看示例，请检查[Project Generators](../project-generation.md){.internal-link target=_blank}，它们都包含已经配置好的Celery。
```
Comment (blt232018) [inline docs/zh/docs/tutorial/background-tasks.md:88]:
```suggestion
要查看示例，请检查 [Project Generators](../project-generation.md){.internal-link target=_blank}，它们都包含已经配置好的 Celery。
```

File: docs/zh/docs/tutorial/background-tasks.md [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+
+但是如果你需要从相同的 FastAPI 应用中访问变量和对象，或者你需要执行小的后台任务(比如发送邮件通知)，你可以简单地使用 `BackgroundTasks` 。
```
Comment (blt232018) [inline docs/zh/docs/tutorial/background-tasks.md:90]:
```suggestion
但是如果你需要在同一个 FastAPI 应用中访问变量和对象，或者需要执行小型的后台任务（比如发送邮件通知），那就可以简单地使用 `BackgroundTasks`。
```

File: docs/zh/docs/tutorial/background-tasks.md [hunk 8]
```diff
@@ -0,0 +1,94 @@
...
+
+## 总结
+
+导入，并在 *路径操作函数* 和依赖项的参数中使用 `BackgroundTasks` 来添加后台任务。
```
Comment (blt232018) [inline docs/zh/docs/tutorial/background-tasks.md:94]:
```suggestion
导入，并在*路径操作函数*和依赖项的参数中使用 `BackgroundTasks` 来添加后台任务。
```

**Review by blt232018 — CHANGES_REQUESTED** [review]

**Comment by tiangolo** [comment]
📝 Docs preview for commit 8a138dc7e9861dc7c667b7805b5f02b038d1708b at: https://6499c7e9da96dc0080f3f275--fastapi.netlify.app

**Comment by tiangolo** [comment]
As this PR had requested changes to be applied but has been inactive for a while, it's now going to be closed. But if there's anyone interested, feel free to create a new PR.

**Pull request closed without merging**